integration/examples/custom-tests/skaffold.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: skaffold/v2beta13
+kind: Config
+build:
+  artifacts:
+  - image: skaffold-example
+test:
+  - image: skaffold-example
```

Comment:
It would be useful to have another test here to exercise the `echo [a,b,c]` logic for both *nix and Windows.

Reply:
Added another custom command instance with `echo ...`.